qutebrowser/mainwindow/tabwidget.py

```diff
@@ -79,6 +79,7 @@ def init_config(self):
         self.setTabPosition(position)
         tabbar.vertical = position in [QTabWidget.West, QTabWidget.East]
         tabbar.setSelectionBehaviorOnRemove(selection_behavior)
+        tabbar.set_font()
```

Comment:
What's the intention of this? This `init_config` block only gets run for changes in the `tabs` section because of the `@config.change_filter('tabs')`. Changing the font is already handled in `TabBar` (`        config_obj.changed.connect(self.set_font)`)